src/components/profession-hero/ProfessionHero.css

```diff
@@ -0,0 +1,215 @@
+.profession {
```

Comment:
@McCambley , do you think it looks better if the width expands with the screen? as long as all the components do it. it would look better than have all the components stop at 1440px; and be surrounded by white page.

Reply:
no prob, i'll tack on the max-widths